cockatrice/src/chatview.cpp

```diff
@@ -374,9 +417,27 @@ void ChatView::appendMessage(QString message, QString sender, UserLevelFlags use
                 while (fullMentionUpToSpaceOrEnd.size());
             }
         }
+        else if (index == highlightWordFirstIndex)
+        {
+            // You have received a valid mention of custom word!!
+            int firstSpace = (message.indexOf(" ") == -1 ? message.size() : message.indexOf(" "));
+            mentionFormat.setBackground(QBrush(getCustomMentionColor()));
+            mentionFormat.setForeground(settingsCache->getChatMentionForeground() ? QBrush(Qt::white) : QBrush(Qt::black));
+            cursor.insertText(message.mid(0, firstSpace), mentionFormat);
+            cursor.setCharFormat(defaultFormat);
+            message = message.mid(firstSpace);
+            QApplication::alert(this);
```

Comment:
I know lots of this is cpypasta form, but doyou think we want to alert when finding these words? As an admin I might want to flag a username, I dont want to be alerted anytime they say something.